Configuration/Policy.yaml

```diff
@@ -2,7 +2,7 @@
 privilegeTargets:
   TYPO3\Flow\Security\Authorization\Privilege\Method\MethodPrivilege:
     TYPO3_FormBuilder_Standard:
-      matcher: 'method((TYPO3\FormBuilder\Controller\StandardController|SimplyAdmire\Neos\FormBuilderBundle\Controller\FormController)->.*Action())'
+      matcher: 'method(TYPO3\FormBuilder\Controller\StandardController|SimplyAdmire\Neos\FormBuilderBundle\Controller\FormController->.*Action())'
```

Comment:
@radmiraal Could you please merge this, it's pretty urgent. Flow 3.3.x totally breaks